[readme]: adds josephakayesi to gitops.md

josephakayesi opened the pull request.

josephakayesi committed:
[readme]: adds josephakayesi to gitops.md

bradmccoydev reviewed (changes requested):
two "h" in https shouldnt be hhttps://

josephakayesi committed:
[fix]: corrects certificate link

adds correction to certifiicate link in gitops.md for joseph akayesi. changes hhttps to https
